### ISSUE TITLE
Incompatibility with gradle 6 (early snapshots)

### ISSUE THREAD

**Issue by yschimke** [diffplug/spotless#440]
Not a concern, just raising it for visibility, feel free to close.

https://circleci.com/gh/square/okhttp/4948?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
FAILURE: Build completed with 18 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':mockwebserver:spotlessKotlin'.
> You must declare outputs or use `TaskOutputs.upToDateWhen()` when using the incremental task API

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================
```

**Comment by yschimke** [comment]
PR branch if its useful to test against.

**Comment by nedtwigg** [comment]
I think this is already fixed in 3.24.2

**Comment by yschimke** [comment]
Thanks
